tests/lib/app_jwt/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(app_jwt_test)
+
+
+FILE(GLOB app_sources src/*.c)
+target_sources(app PRIVATE ${app_sources})
+
+target_sources(app
+  PRIVATE
+  ${ZEPHYR_NRF_MODULE_DIR}/lib/app_jwt/app_jwt.c
+  )
+
+target_include_directories(app
+  PRIVATE
+  ${ZEPHYR_NRF_MODULE_DIR}/subsys/net/lib/nrf_cloud/include/
+  ${ZEPHYR_NRF_MODULE_DIR}/../modules/crypto/tinycrypt/lib/include/
```

Comment:
this should use the proper variable for the path, not use another module directory and doing a relative path to a different module, but that's not the main issue, the main issue is that this is adding something using tinycrypt which is deprecated and has been removed from zephyr

Reply:
removed, this wasn't needed anymore